### PR TITLE
Create ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by luja-c8y

Ticket: [MTM-60214](https://cumulocity.atlassian.net/browse/MTM-60214)
Original PR: [7721](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7721)

[MTM-60214]: https://cumulocity.atlassian.net/browse/MTM-60214?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: As a developer, I want to implement a generic datapoints export component (#7129) [GRAFT][release/cd]  (#7721)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]: The Claude AI suggested to use the following title:
```suggestion
title: New datapoint export selector
```

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60214
+version: 1021.14.0
+---
+As a developer, I want to implement a generic datapoints export component (#7129) [GRAFT][release/cd]  (#7721)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]: The Claude AI suggested to use the following body:
```suggestion
To enable users to easily export datapoints from Cumulocity IoT, a new generic datapoints export component has been implemented. This component allows users to select the devices, the time range and the format for the datapoints export. The exported file will then be generated asynchronously and can be downloaded by the user once it is ready. This new component makes it much simpler for users to get the device data they need out of Cumulocity IoT in their desired format.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md (hunk 1 above)
Comment (luja-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]: In our repo this component is called: `datapoints-export-selector` so I would go with:
```suggestion
title: Datapoints export selector
```

or

```suggestion
title: Datapoints export selector component
```

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md (hunk 2 above)
Comment (luja-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]:
```suggestion
To allow users to easily export datapoints from Cumulocity IoT, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoints export. Depending on the size of the datapoints records to be exported, the exported file is then generated and downloaded directly or emailed when ready.  This new component makes it much simpler for users to get the device data they need out of Cumulocity IoT in their desired format.
```
or without the last sentence which seems unnecessary:

```suggestion
To allow users to easily export datapoints from Cumulocity IoT, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoints export. Depending on the size of the datapoints records to be exported, the exported file is then generated and downloaded directly or emailed when ready.
```

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60214
+version: 1021.14.0
+---
+To allow users to easily export datapoints from Cumulocity IoT, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoints export. Depending on the size of the datapoints records to be exported, the exported file is then generated and downloaded directly or emailed when ready.  This new component makes it much simpler for users to get the device data they need out of Cumulocity IoT in their desired format.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]:
```suggestion
To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. This new component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60214
+version: 1021.14.0
+---
+To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. This new component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]:
```suggestion
To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export selector has been implemented. This new component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. The datapoint export selector component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```

**Review by rahulc8y — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: New datapoint export selector
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]:
```suggestion
title: New datapoint export component
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60214
+version: 1021.14.0
+---
+To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export selector has been implemented. This new component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. The datapoint export selector component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md]:
```suggestion
To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export selector has been implemented. This new component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. The datapoint export component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```

**Pull request merged**